docs/source/developer_resources.rst

```diff
@@ -339,6 +339,44 @@ Best Practices
 8. Use the “DO NOT MERGE” label for Pull Requests that should not be merged
 9. When PR has been reviewed and approved, move the ticket/issue to the 'Ready to Deploy to Dev' box in the GitHub project tracker.
 
+Git Naming Conventions
+----------------------
+
+Commit messages should follow the format of
+
+.. code-block:: console
+
+    <type>[optional scope]: <description>
+
+    [optional body]
+
+:code:`type` must be one of the following:
+
+- **docs**: Changes to the documentation (e.g. improving docstring, updating this file, etc)
+- **feat**: Adds a new feature
+- **fix**: A bug fix
+- **refactor**: Changes that don't fix a bug or add a new feature
+- **style**: Changes that don't affect the meaning of code (e.g. whitespace)
+- **test**: Adding or correcting tests
```

Comment:
Looking for feedback on these options, as well as their clarity (ie if you had to write a commit message for the last change you made, is this clear enough so that you could determine what to use?).

Some other ones people use are:
- ci
- build
- chore
- perf

Reply:
The six that you've got seem to cover the bases.  I know we've used `chore` a lot in the past, but `style` makes sense to me for issues like flake8 and linting